### DISCUSSION TITLE
Adding electric field EFIELD

### DISCUSSION THREAD

**Discussion by bp432** [Chengcheng-Xiao/VASP2WAN90_v2_fix#22] [Q&A]
Dear Chengcheng,

I hope you are doing well! Thanks for sharing your patch, which is really helpful. I noticed an issue when running calculations under electric field with EFIELD and PBE0, I got the same total energy for EFIELD = 0 and EFIELD = 1 whenever setting LWANNIER90 = T. However, switch this off leads to different total energies. Could you please help me check this? Thanks in advance!

Best wishes,
Bo Peng

**Comment by Chengcheng-Xiao** [comment]
Hum, I've not experienced such behavior. Are you running v5.4.4? 

The only thing I reckon might be relevant is symmetry. Setting `LWANNIER90=.T.` automatically turns off symmetry at the end of the calculation. Also, since Wannier routines are only called after the electronic minimization, it might be worth a while to check the convergence as well as the energy of the last step (before the interface is called) compared to the ones without `LWANNIER90=.T.`.
